webhook-bridge/svix-webhook-bridge.example3.yaml

```
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
+
+# Global svix settings.
+# A svix client is needed by each sender.
+# Each sender can override any of these fields by defining a `svix_client` section.
+global_svix_client:
+  token: "XXXXX"
+  options:
+    debug: true
+    server_url: http://example.com/
+
+# Webhook Senders configure an input and (optional) transformation.
+# Use transformations to conform the payload we get as input to the expected shape (app_id, message, etc).
+# When transformation is left unset, the payload is used as-is.
+senders:
+  - name: "rabbit-local"
+    input:
+      type: "rabbitmq"
+      queue: "local"
+      uri: "amqp://example.com/%2f"
+      requeue_on_nack: true
+    transformation: |
+      handler = (x) => ({ app_id: "app_1234", message: { eventType: "foo.bar", payload: x }})
+    # Can optionally specify`svix_client` here, but can be omitted entirely if the global is set.
+
+  - name: "rabbit-local2"
+    input:
+      type: "rabbitmq"
+      queue: "local2"
+      uri: "amqp://example.com/%2f"
+      requeue_on_nack: true
+    transformation: |
+      handler = (x) => ({ app_id: "app_1234", message: { eventType: "foo.bar", payload: x }})
+    # Output can optionally override `global_svix_client`
+    svix_client:
+      # the options.debug and options.server_url are inherited from the global
+      token: "YYYY"
+
+# Webhook Receivers each register an HTTP endpoint that maps to an output (ie. a message queue or what have you).
+receivers:
+  - path: "/from-svix"
+    verification:
+      # Use the standard svix signature verification scheme.
+      type: "svix"
+      endpoint_secret: "whsec_XXXXXXXXXXX="
```

```suggestion
    input:
      # Use the standard svix signature verification scheme.
      type: "svix"
      endpoint_secret: "whsec_XXXXXXXXXXX="
```
I think sticking with the input/output is potentially clear regardless of the other thing.

---

We can also have a "shortcut" potentially by having:
```yaml
    input:
      # Use the standard svix signature verification scheme.
      type: "svix-webhook"
      endpoint_secret: "whsec_XXXXXXXXXXX="
```
So we can support both the above and this one and this is jut a short version for the above. For example.

---

I included this idea in `.example5.receivers.yaml` but kept the `verification` terminology since I think `authentication` speaks more to tokens and HTTP Authorization headers and signature verification feels sort of different.

I could easily see a case for needing proper HTTP authorization checks in addition to or instead of signature verification. Better to not conflate.